GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -240,24 +427,125 @@ gdjs.HowlerSoundManager.prototype._storeSoundInArray = function (arr, sound) {
   return sound;
 };
 
+/**
+ * Creates a new gdjs.HowlerSound using preloaded/cached Howl instances.
+ * @param {string} soundName The name of the file or resource to play.
+ * @param {boolean} isMusic True if a music, false if a sound.
+ * @returns {gdjs.HowlerSound}
+ */
+gdjs.HowlerSoundManager.prototype.createHowlerSound = function (
+  soundName,
+  isMusic
+) {
+  const soundFile = this._getFileFromSoundName(soundName);
+  const cacheContainer = isMusic ? this._loadedMusics : this._loadedSounds;
+
+  if (!cacheContainer.hasOwnProperty(soundFile)) {
+    //@ts-ignore Ts doesn't know about Howler
+    cacheContainer[soundFile] = new Howl({
+      src: [soundFile],
+      preload: true,
+      html5: isMusic,
+      onplayerror: (_, error) =>
+        console.error("Can't play a sound. Error is: ", error),
+      onloaderror: (_, error) =>
+        console.error('Error while loading sound file: ', error),
+    });
+  }
+  return new gdjs.HowlerSound(cacheContainer[soundFile]);
+};
+
+/**
+ * Preloads a sound or a music in memory.
+ * @param {string} soundName The name of the file or resource to preload.
+ * @param {boolean} isMusic True if a music, false if a sound.
+ */
+gdjs.HowlerSoundManager.prototype.loadAudio = function (soundName, isMusic) {
+  const soundFile = this._getFileFromSoundName(soundName);
+  const cacheContainer = isMusic ? this._loadedMusics : this._loadedSounds;
+
+  //@ts-ignore Ts doesn't know about Howler
+  cacheContainer[soundFile] = new Howl({
```

Comment:
Do you think we should add a condition like `if (!cacheContainer.hasOwnProperty(soundFile)) {`, like for `createHowlerSound`, as it would be not necessary to erase the existing `Howl` if there is already one in the cacheContainer (and that can be already used by some `gdjs.HowlerSound`)?